stock_move_packaging_qty/tests/test_stock_move_packaging_qty.py

```diff
@@ -59,3 +66,19 @@ def test_product_uom_qty_change(self):
             move_f.product_packaging_id = self.packaging
             self.assertEqual(move_f.product_uom_qty, 5)
             self.assertEqual(move_f.product_packaging_qty, 1)
+        picking = picking_f.save()
+        self.assertEqual(picking.state, "draft")
+        picking.action_set_quantities_to_reservation()
```

Comment:
Assign picking before this line

```suggestion
        picking.action_assign()
        picking.action_set_quantities_to_reservation()
```